sdcm/sct_config.py

```diff
@@ -1128,7 +1128,29 @@ class SCTConfiguration(dict):
              help="Limit severity level for event types"),
 
         dict(name="scylla_rsyslog_setup", env="SCT_SCYLLA_RSYSLOG_SETUP", type=boolean,
-             help="Configure rsyslog on Scylla nodes to send logs to monitoring nodes")
+             help="Configure rsyslog on Scylla nodes to send logs to monitoring nodes"),
+
+        dict(name="data_device", env="SCT_DATA_DEVICE", type=str,
```

Comment:
let's rename it to `data_volume_device_type`

Reply:
done